specification/cognitiveservices/data-plane/SpellCheck/V1.0/SpellCheck.json

```diff
@@ -0,0 +1,468 @@
+            "description": "A unique name of the device that the request is being made from. Generate a unique value for each device (the value is unimportant). The service uses the ID to help debug issues and improve the quality of corrections.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "DocId",
+            "in": "query",
+            "description": "A unique ID that identifies the document that the text belongs to. Generate a unique value for each document (the value is unimportant). The service uses the ID to help debug issues and improve the quality of corrections.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "mkt",
+            "x-ms-client-name": "Market",
+            "in": "query",
+            "description": "The market where the results come from. You are strongly encouraged to always specify the market, if known. Specifying the market helps Bing route the request and return an appropriate and optimal response. This parameter and the cc query parameter are mutually exclusive—do not specify both.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "SessionId",
+            "in": "query",
+            "description": "A unique ID that identifies this user session. Generate a unique value for each user session (the value is unimportant). The service uses the ID to help debug issues and improve the quality of corrections",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "SetLang",
+            "in": "query",
+            "description": "The language to use for user interface strings. Specify the language using the ISO 639-1 2-letter language code. For example, the language code for English is EN. The default is EN (English). Although optional, you should always specify the language. Typically, you set setLang to the same language specified by mkt unless the user wants the user interface strings displayed in a different language. This parameter and the Accept-Language header are mutually exclusive—do not specify both. A user interface string is a string that's used as a label in a user interface. There are few user interface strings in the JSON response objects. Also, any links to Bing.com properties in the response objects apply the specified language.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "UserId",
+            "in": "query",
+            "description": "A unique ID that identifies the user. Generate a unique value for each user (the value is unimportant). The service uses the ID to help debug issues and improve the quality of corrections.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "Mode",
+            "in": "formData",
```

Comment:
You doc says that mode is `query`:
https://dev.cognitive.microsoft.com/docs/services/5f7d486e04d2430193e1ca8f760cd7ed/operations/56e73036cf5ff81048ee6727

Could you confirm it's formData?

Reply:
Yes. We support only "Post" http methods through this SDK and "Mode" is a post body parameter. The documentation appears to be inaccurate.